worker/workercmd/worker_linux.go

```diff
@@ -29,9 +29,11 @@ type GardenBackend struct {
 	Bin        string    `long:"bin"        description:"Path to a garden backend executable (non-absolute names get resolved from $PATH)."`
 	Config     flag.File `long:"config"     description:"Path to a config file to use for the Garden backend. Guardian flags as env vars, e.g. 'CONCOURSE_GARDEN_FOO_BAR=a,b' for '--foo-bar a --foo-bar b'."`
 	DNSServers []string  `long:"dns-server" description:"DNS server IP address to use instead of automatically determined servers. Can be specified multiple times."`
-	DNS            DNSConfig     `group:"DNS Proxy Configuration" namespace:"dns-proxy"`
+	DNS        DNSConfig `group:"DNS Proxy Configuration" namespace:"dns-proxy"`
 
 	RequestTimeout time.Duration `long:"request-timeout" default:"5m" description:"How long to wait for requests to Garden to complete. 0 means no timeout."`
```

Comment:
This flag is now responsible for more than just requests to Garden, now

Might it make sense to split it up into a separate flag?

Reply:
I believe there's a larger story at moving all the containerd flags out of the `GARDEN` namespace at https://github.com/concourse/concourse/issues/5407

Reply:
True, but I guess my point is that the flag is being used for two things - one is related to garden (and makes sense to be in the `GARDEN` namespace), while the other is containerd specific (and doesn't). But maybe we can address this in #5407

Reply:
Yes, the flag is currently used for both `containerd` & `garden` and the reason was to limit scope to this PR to having a means of configuring max container limit for `containerd`. 
The scope of #5407 is to look at all flags that are used commonly between the 2 runtimes and dealing with them en masse.